package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "generator-ship-components-boilerplate",
+  "version": "0.1.0",
+  "description": "simple react boilerplate for the SHIP team",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/generator-ship-components-boilerplate/issues"
+  },
+  "homepage": "https://github.com/ship-components/generator-ship-components-boilerplate#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/generator-ship-components-boilerplate.git"
+  },
+  "scripts": {
+    "prepare": "gulp prepublish",
+    "test": "gulp",
+    "travis": "npm i trevor -g && trevor"
```

Comment:
We shouldn't install this globally in the script. I might not want to install this globally or I might have to run sudo to install it globally.